doc/external-libs/trilinos.html

```diff
@@ -116,6 +116,27 @@
 
     <h5>BLAS and LAPACK</h5>
 
+    <p style="color: red">
+      Note: At the time of writing (November 2016) Trilinos cannot link against 
+      LAPACK 3.6.0 or later, due to two symbols that were deprecated in LAPACK 
+      3.6.0. To fix this, edit the Trilinos file 
```

Comment:
To clarify, in fact I think it is the following (although I am leaving my area of comfort here!). 

I believe the symbols are still technically only deprecated (they now live at e.g. https://github.com/Reference-LAPACK/lapack/blob/master/SRC/DEPRECATED/sggsvd.f). However, LAPACK introduced a BUILD_DEPRECATED cmake option https://github.com/Reference-LAPACK/lapack/commit/49b7de3e364a54cdb4b082c937ddda8691a49326 and (I am inferring here) my sysadmins built LAPACK with BUILD_DEPRECATED=OFF (the default).

If that makes sense, then I can either update this text to include this technical caveat (presumably if BUILD_DEPRECATED=ON then Trilinos will build fine) or edit it to "were removed"?